utils/src/main/java/org/jclouds/karaf/utils/EnvHelper.java

```diff
@@ -92,6 +98,22 @@ public static String getComputeCredential(String credential) {
     }
 
     /**
+     * Extracts the credential value from the Google Cloud credentials json file.
+     * @param jsonFile
+     * @return
+     */
+    public static String getGoogleCredentialFromJsonFile(String jsonFile) {
```

Comment:
Live tests might read a file but karaf expected the actual credentals (the pem contents) and failed to parse a multi-line value. When adding the logic to read a file I added the one to read the json form, which is the Google's recommended one and also the one used in the jclouds-examples.